airgap/client.go

```diff
@@ -90,9 +101,7 @@ func (c *clientImpl) SignTx(tx *Transaction, privateKey *ecdsa.PrivateKey) (*Tra
 }
 
 func (c *clientImpl) GenerateProofOfPossessionSignature(privateKey *ecdsa.PrivateKey, address *common.Address) ([]byte, error) {
-	msg := accounts.TextHash(address.Bytes())
-	signature, err := c.Sign(msg, privateKey)
-	return signature, err
+	return c.SignWithPrefix(address.Bytes(), privateKey)
```

Comment:
can you just do the following and not have to define `SignWithPrefix`? not sure if it will be used elsewhere?
```suggestion
      return c.SignHash(accounts.TextHash(address.Bytes()), privateKey)
```

Reply:
I think this makes it more explicit that `TextHash` is doing the prefix transformation

Reply:
and that the pop needs the prefix